communication/slack-config/channels.yaml

```diff
@@ -142,6 +142,7 @@ channels:
   - name: kube-spawn
   - name: kube-state-metrics
   - name: kubeadm
+  - name: kubeadm-china
```

Comment:
nit: consider `kubeadm-zh` (the language) or `kubeadm-cn` (the region) to fit in the current naming style